core/src/main/java/org/apache/struts2/config/StrutsBeanSelectionProvider.java

```diff
@@ -418,6 +419,8 @@ public void register(ContainerBuilder builder, LocatableProperties props) {
         /** Checker is used mostly in interceptors, so there be one instance of checker per interceptor with Scope.PROTOTYPE **/
         alias(ExcludedPatternsChecker.class, StrutsConstants.STRUTS_EXCLUDED_PATTERNS_CHECKER, builder, props, Scope.PROTOTYPE);
         alias(AcceptedPatternsChecker.class, StrutsConstants.STRUTS_ACCEPTED_PATTERNS_CHECKER, builder, props, Scope.PROTOTYPE);
+        alias(NotExcludedAcceptedPatternsChecker.class, StrutsConstants.STRUTS_NOT_EXCLUDED_ACCEPTED_PATTERNS_CHECKER
+                , builder, props, Scope.SINGLETON);
```

Comment:
Strange formatting

Reply:
IntelliJ has a vertical line indicating suggested line length limit. I always press enter on it to not violate it. Do you want me to keep all in same line? I can if so :)

Reply:
I think it is about comma in the begging of the line.